docs/examples/1d_hist/1d_profile.py

```diff
@@ -0,0 +1,31 @@
+"""
+1d Profile
+==========
+
+Plot a 1d mean histogram (profile plot).
+"""
+
+###
+import boost_histogram as bh
+from plothist import plot_error_hist
+import matplotlib.pyplot as plt
+
+fig, ax = plt.subplots()
+
+# Regular axis with 3 bins from -1 to 1
+axis = bh.axis.Regular(3, -1, 1)
+
+data = [-0.5, -0.5, 0.0, 0.0, 0.5, 0.5]
```

Comment:
Add a small comment to explain what the data will look like, as you did with `axis`.